libraries/MotorController/motorControls.h

```diff
@@ -0,0 +1,49 @@
+/*
+ *Direction 
+ *0 - Clockwise - Negative
+ *1 - Counter-Clockwise - Positive
+ */
+#ifndef motorControls_h
+#define motorControls_h
+#include <Arduino.h>
+#include "MotorControllerConstants.h"
+
+
+class MotorControls
+{
+
+private:
+	int positionDifference;
+
+	// Step length functions. Full step default
+	void fullStep();
+	void halfStep();
+	void quarterStep();
+	void eighthStep();
+	
+	// Pass in number of steps to desired position and the direction
+	// param [in] moveSteps 		  - Number of steps between current position and desired new position
+	// param [in] direction 		  - Enum indicates direction (reverse: 0/forward: 1/unknown: 2/total: 3) of travel
+	// param [in] stepSize (Optional) - Number defining size of step (default: 0)
+    void moveMotor(int moveSteps, enum direction, int stepSize = 0);
+	void pinSetup(); //Setup the pins per the MotorControllerConstants.h file
+    void turnMotorOn();
```

Comment:
Fix the alignment issues

Reply:
The alignment issue is on git side. I prefer alignment in my code too...